charts/linkerd2/README.md

```diff
@@ -113,6 +114,7 @@ their default values.
 | `global.linkerdNamespaceLabel`              | Control plane label. Do not edit                                                                                                                                                      | `linkerd.io/is-control-plane` |
 | `global.linkerdVersion`                     | Control plane version                                                                                                                                                                 | latest version                       |
 | `global.namespace`                          | Control plane namespace                                                                                                                                                               | `linkerd`                            |
+| `global.prometheusUrl`     | URL of prometheus instance to perform queries, used by the `public-api`                                                                                                                    |                               |
```

Comment:
Similar to the Grafana change above, thoughts on adding "external" so that it is obvious this is a Prometheus instance that exists outside of the Linkerd installation?

```suggestion
| `global.prometheusUrl`     | URL of external prometheus instance to perform queries, used by the `public-api`                                                                                                                    |                               |
```